src/commands/math/advancedSymbols.ts

```diff
@@ -737,3 +737,227 @@ LatexCmds.measuredangle = bindVanillaSymbol(
+LatexCmds['κ'] = () =>
+  new VanillaSymbol('\\kappa ', h.entityText('&kappa;'), 'kappa');
+LatexCmds['ι'] = () =>
+  new VanillaSymbol('\\iota ', h.entityText('&iota;'), 'iota');
+LatexCmds['θ'] = () =>
+  new VanillaSymbol('\\theta ', h.entityText('&theta;'), 'theta');
+LatexCmds['η'] = () =>
+  new VanillaSymbol('\\eta ', h.entityText('&eta;'), 'eta');
+LatexCmds['ζ'] = () =>
+  new VanillaSymbol('\\zeta ', h.entityText('&zeta;'), 'zeta');
+LatexCmds['δ'] = () =>
+  new VanillaSymbol('\\delta ', h.entityText('&delta;'), 'delta');
+LatexCmds['γ'] = () =>
+  new VanillaSymbol('\\gamma ', h.entityText('&gamma;'), 'gamma');
+LatexCmds['β'] = () =>
+  new VanillaSymbol('\\beta ', h.entityText('&beta;'), 'beta');
+LatexCmds['α'] = () =>
+  new VanillaSymbol('\\alpha ', h.entityText('&alpha;'), 'alpha');
+LatexCmds['ϕ'] = () =>
+  new VanillaSymbol('\\phi ', h.entityText('&phi;'), 'phi');
+LatexCmds['φ'] = () =>
+  new VanillaSymbol('\\varphi ', h.entityText('&varphi;'), 'varphi');
+LatexCmds['ϵ'] = () =>
+  new VanillaSymbol('\\epsilon ', h.entityText('&epsilon;'), 'epsilon');
+LatexCmds['ε'] = () =>
+  new VanillaSymbol(
+    '\\varepsilon ',
+    h.entityText('&varepsilon;'),
+    'varepsilon'
+  );
+LatexCmds['ϖ'] = () =>
+  new VanillaSymbol('\\varpi ', h.entityText('&varpi;'), 'varpi');
+LatexCmds['ς'] = () =>
+  new VanillaSymbol('\\varsigma ', h.entityText('&varsigma;'), 'varsigma');
+LatexCmds['ϑ'] = () =>
+  new VanillaSymbol('\\thetasym ', h.entityText('&thetasym;'), 'thetasym');
+LatexCmds['υ'] = () =>
+  new VanillaSymbol('\\upsi ', h.entityText('&upsi;'), 'upsi');
+LatexCmds['ϝ'] = () =>
+  new VanillaSymbol('\\digamma ', h.entityText('&digamma;'), 'digamma');
+LatexCmds['ϰ'] = () =>
+  new VanillaSymbol('\\varkappa ', h.entityText('&varkappa;'), 'varkappa');
+LatexCmds['ϱ'] = () =>
+  new VanillaSymbol('\\varrho ', h.entityText('&varrho;'), 'varrho');
+LatexCmds['λ'] = () =>
+  new VanillaSymbol('\\lambda ', h.entityText('&lambda;'), 'lambda');
+LatexCmds['Υ'] = () =>
+  new VanillaSymbol('\\Upsilon ', h.entityText('&Upsilon;'), 'Upsilon');
+LatexCmds['∀'] = () =>
+  new VanillaSymbol('\\forall ', h.entityText('&forall;'), 'forall');
+LatexCmds['Ω'] = () =>
+  new VanillaSymbol('\\Omega ', h.entityText('&Omega;'), 'Omega');
+LatexCmds['Φ'] = () =>
+  new VanillaSymbol('\\Phi ', h.entityText('&Phi;'), 'Phi');
+LatexCmds['Σ'] = () =>
+  new VanillaSymbol('\\Sigma ', h.entityText('&Sigma;'), 'Sigma');
+LatexCmds['Π'] = () => new VanillaSymbol('\\Pi ', h.entityText('&Pi;'), 'Pi');
+LatexCmds['Ξ'] = () => new VanillaSymbol('\\Xi ', h.entityText('&Xi;'), 'Xi');
+LatexCmds['Λ'] = () =>
+  new VanillaSymbol('\\Lambda ', h.entityText('&Lambda;'), 'Lambda');
+LatexCmds['Θ'] = () =>
+  new VanillaSymbol('\\Theta ', h.entityText('&Theta;'), 'Theta');
+LatexCmds['Δ'] = () =>
+  new VanillaSymbol('\\Delta ', h.entityText('&Delta;'), 'Delta');
+LatexCmds['Γ'] = () =>
+  new VanillaSymbol('\\Gamma ', h.entityText('&Gamma;'), 'Gamma');
```

Comment:
All greek letters already have a command defined in basicsymbols. Bind the unicode representations similar to those as you've done below